src/main/java/appeng/util/ColorPickHelper.java

```diff
@@ -0,0 +1,20 @@
+package appeng.util;
+
+import appeng.core.localization.GuiColors;
+
+public class ColorPickHelper {
+
+    public static GuiColors selectColorFromThreshold(float threshold) {
+        GuiColors color = null;
+        if (threshold <= 25) {
+            color = GuiColors.CraftConfirmPercent25;
+        } else if (threshold <= 50) {
+            color = GuiColors.CraftConfirmPercent50;
+        } else if (threshold <= 75) {
+            color = GuiColors.CraftConfirmPercent75;
+        } else if (threshold <= 100) {
```

Comment:
Better to always return a color
```suggestion
        } else {
```